package.json

```diff
@@ -28,11 +28,25 @@
     "babel-polyfill": "^6.23.0",
     "chai": "^3.5.0",
     "eslint": "^3.19.0",
+    "eslint-config-prettier": "^2.9.0",
     "eslint-config-standard": "^7.1.0",
+    "eslint-plugin-prettier": "^2.6.0",
     "eslint-plugin-promise": "^3.5.0",
     "eslint-plugin-standard": "^2.1.1",
+    "husky": "^0.14.3",
     "iron-node": "^3.0.18",
+    "lint-staged": "^7.0.0",
     "mocha": "^3.2.0",
+    "prettier": "^1.10.2",
     "sinon": "^2.1.0"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run test",
+      "pre-push": "npm run test"
+    }
+  },
+  "lint-staged": {
+    "*.js": ["npm run lint", "git add"]
```

Comment:
Are we sure `git add` here only adds the modified staged files? e.g. I have 3 staged 3 unstaged and I commit, it will lint the 3 staged files, but perhaps `git add` will add the 3 unstaged to the commit too, which shouldn't happen